app/src/main/java/dev/msfjarvis/lobsters/ui/main/MainActivity.kt

```diff
@@ -58,6 +58,8 @@ fun LobstersApp() {
   val hottestPosts = viewModel.posts.collectAsLazyPagingItems()
   val savedPosts by viewModel.savedPosts.collectAsState()
 
+  viewModel.getSavedPosts()
```

Comment:
This is a leaky abstraction, we should have this be done by the VM itself.

Reply:
Yeah, we can have this in viewModel's init block.

Reply:
Still needs fixing.